recipes/list_items/contact_info/contact_info.stories.js

```diff
@@ -287,8 +302,8 @@ Variants.parameters = {
       <template #subtitle>+1 (415) 123-4567</template>
       <template #right>
         <div class="d-d-flex d-ai-center d-m16">
-          <icon-chat class="d-m4" />
-          <icon-menu-horizontal class="d-m4" />
+          <dt-icon name="webchat" class="d-m4" />
+          <dt-icon name="more-horizontal" class="d-m4" />
```

Comment:
Since these are clearly actionable, why are they not wrapped in a `button`?

Reply:
You're right, wrapping them in button